CHANGELOG.unreleased.md

```diff
@@ -24,5 +25,6 @@
 >
 > Rule of thumb: add packages on top.
 
-- xo-server v5.51.0
-- xo-web v5.51.0
+- xo-server-cloud v0.3.0
+- xo-server v5.52.0
+- xo-web v5.52.0
```

Comment:
`xo-server` and `xo-web`'s versions are already bumped on each release. They're the only packages you don't need to take care of.